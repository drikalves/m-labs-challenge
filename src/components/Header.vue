<template>
  <header class="header">
    <div class="bars">
      <img src="../assets/images/Bars.png" alt="Bars">
    </div>
    <div class="avatar-menu">
      <img src="../assets/images/Avatar.png" alt="Avatar">
    </div>
  </header>
</template>

<style lang="scss">
.header {
  align-items: center;
  background: #ffffff;
  display: flex;
  padding-bottom: 18px;
  padding-left: 20px;
  padding-top: 18px;
  padding-right: 20px;
  justify-content: space-between;
}
</style>
