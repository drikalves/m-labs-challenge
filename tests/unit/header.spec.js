import { shallowMount } from '@vue/test-utils';
import Header from '@/components/Header.vue';

describe('Header.vue', () => {
  it('to match with snapshot', () => {
    const wrapper = shallowMount(Header);
    expect(wrapper.html()).toMatchSnapshot();
  });
});
