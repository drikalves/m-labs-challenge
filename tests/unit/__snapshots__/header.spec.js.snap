// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header.vue to match with snapshot 1`] = `
<header class="header"><i class="header__bars fas fa-bars fa-2x"></i> <img src="../assets/images/mlabs-logo.png" alt="logo" class="header__logo">
  <div class="header__user"><img src="../assets/images/Avatar.svg" alt="avatar" class="header__user-avatar">
    <p class="header__user-name">Anselmo Carlos</p>
  </div>
</header>
`;
